guard.cpp

```diff
@@ -18,7 +18,8 @@ void guardList()
     std::cout << "ID       | ERROR    |  Type  | Path " << std::endl;
     for (const auto& elem : records)
     {
-        if (elem.errType != GARD_Reconfig)
+        if ((elem.errType != GARD_Reconfig) &&
```

Comment:
guard tool does not have any requirement to show the invalid record (resolved record)

Reply:
We should not show any invalid recordId in tool. Even in reconfig path the guard recordId is set to FFFFFFFF. And displaying an invalid record ID doesnt make much sense.

Reply:
Reconfig and Resolved (Invalid) are different right?
It will be greater if you gave the option to show resolved records but, it's your wish.

Reply:
The issue is for reconfig type guard record the recordId gets updated to FF..FF. Not sure if we should think as its resolved ?

Reply:
I am not talking for reconfig record (fake guard record), just checking whether the tool has a requirement to show the resolved record (0xFFFFFFFF)?
If no, do you need to check with the tool user?

Reply:
We will show the resolved guard records. Added option for that.